client/landing/jetpack-cloud/components/filter-bar/activity-type-selector.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent, useRef, ChangeEvent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormLabel from 'components/forms/form-label';
+import Popover from 'components/popover';
+
+interface ActivityCount {
+	count: number;
+	key: string;
+	name: string;
+}
+
+interface Props {
+	groups: string[];
+	onGroupsChange: ( newGroups: string[] ) => void;
+	isVisible: boolean;
+	onClick: () => void;
+	onClose: () => void;
+	activityCounts: ActivityCount[];
+}
+
+const ActivityTypeSelector: FunctionComponent< Props > = ( {
+	isVisible,
+	onClick,
+	onClose,
+	activityCounts,
+	groups,
+	onGroupsChange,
+} ) => {
+	const translate = useTranslate();
+	const buttonRef = useRef();
+
+	const onChange = ( { target: { checked, name } }: ChangeEvent< HTMLInputElement > ) => {
+		onGroupsChange(
+			( checked ? [ ...groups, name ] : groups.filter( ( key ) => key !== name ) ).filter(
+				( value, index, self ) => self.indexOf( value ) === index
+			)
+		);
+	};
+
+	return (
+		<>
+			<Button ref={ buttonRef } compact onClick={ onClick }>
+				{ translate( 'Activity type' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Activity Type' )` *ES Score: 10*